test/gtest/ucp/test_ucp_am.cc

```diff
@@ -343,11 +343,14 @@ class test_ucp_am_nbx : public test_ucp_am_base {
 
     test_ucp_am_nbx()
     {
-        m_dt          = ucp_dt_make_contig(1);
-        m_am_received = false;
-        m_rx_dt       = ucp_dt_make_contig(1);
-        m_rx_buf      = NULL;
-        m_rx_memh     = NULL;
+        m_dt                    = ucp_dt_make_contig(1);
```

Comment:
why need to change base class? Can just add new test to `test_ucp_am_nbx_rndv`

Reply:
you can define another small handler, like `am_data_drop_rndv_cb`

Reply:
done